pkg/sfu/connectionquality/connectionstats.go

```diff
@@ -40,7 +40,7 @@ type ConnectionStatsParams struct {
 	IncludeRTT                bool
 	IncludeJitter             bool
 	GetDeltaStats             func() map[uint32]*buffer.StreamStatsWithLayers
-	GetDeltaStatsOverridden   func() map[uint32]*buffer.StreamStatsWithLayers
+	GetDeltaStatsSender       func() map[uint32]*buffer.StreamStatsWithLayers
 	GetLastReceiverReportTime func() time.Time
 	GetTotalPacketsSent       func() uint64
```

Comment:
would an interface or maybe 2 make sense here?

Reply:
Yeah will look at that Paul. It is a bit whacky based on publisher or subscribers using it, but can dummy impl methods to satisfy the interface. Will make a separate PR for that.